index.html

```diff
@@ -7,48 +7,167 @@
-    don't hesitate to contact me.</p><br>
-        <ul id="links">
-          <li>
-            <a class="link" href="https://github.com/tajemouti" rel="noopener noreferrer" target="_blank">
-            <img src="./svg/github.svg" alt="GitHub">
-            </a>
-          </li>
-          <li>
-            <a class="link" href="https://www.linkedin.com/in/ajrass" rel="noopener noreferrer" target="_blank">
-              <img src="./svg/linkedin.svg" alt="LinkedIn">
-            </a>
-          </li>
-          <li>
-            <a class="link" href="https://wellfound.com/u/ajrass-tajemouti" rel="noopener noreferrer" target="_blank">
-              <img src="./svg/wellfound.svg" alt="WellFound">
-            </a>
-          </li>
-          <li>
-            <a class="link" href="https://twitter.com/AjrassTajemouti" rel="noopener noreferrer" target="_blank">
-              <img src="./svg/twitter.svg" alt="Twitter">
-            </a>
-          </li>
-          <li>
-            <a class="link" href="https://medium.com/@tajemouti/" rel="noopener noreferrer" target="_blank">
-              <img src="./svg/medium.svg" alt="Medium">
-            </a>
-          </li>
+    <section class="headline background">
+      <h1 class="hello">Hey There.<br>I'm Ajrass.</h1>
+      <h2 class="iam">I'm a Software Developer</h2>
+      <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+      <ul class="social">
+        <li>
+          <a class="link" href="https://github.com/tajemouti" rel="noopener noreferrer" target="_blank">
+            <img src="default/SVG/IconGitHub.svg" alt="GitHub">
+          </a>
+        </li>
+        <li>
+          <a class="link" href="https://www.linkedin.com/in/ajrass" rel="noopener noreferrer" target="_blank">
+            <img src="default/SVG/Icon Linkedin.svg" alt="LinkedIn">
+          </a>
+        </li>
+        <li>
+          <a class="link" href="https://wellfound.com/u/ajrass-tajemouti" rel="noopener noreferrer" target="_blank">
+            <img src="default/SVG/IconAngelist.svg" alt="WellFound">
+          </a>
+        </li>
+        <li>
+          <a class="link" href="https://twitter.com/AjrassTajemouti" rel="noopener noreferrer" target="_blank">
+            <img src="default/SVG/IconTwitter.svg" alt="Twitter">
+          </a>
+        </li>
+        <li>
+          <a class="link" href="https://medium.com/@tajemouti/" rel="noopener noreferrer" target="_blank">
+            <img src="default/SVG/IconMedium.svg" alt="Medium">
+          </a>
+        </li>
+      </ul>
+    </section>
+    
+    <section class="work">
+      <h1 class="title">My Recent Works</h1>
```

Comment:
Kindly, I suggest you use a `h2` tag instead of the `h1`, because `h1` can be used only for First Heading. 😄